nornir/plugins/inventory/helpers/filters.py

```diff
@@ -0,0 +1,79 @@
+import copy
```

Comment:
I think we should consider this core functionality so I'd place it under `nornir/core/filter.py`